slam-libraries/viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -179,6 +194,8 @@ void SLAMServiceImpl::CreateMap() {
     mapBuilder.map_builder_->pose_graph()->RunFinalOptimization();
     PaintMap(mapBuilder.map_builder_, this->path_to_map + "/images", "0");
```

Comment:
Will we eventually get rid of the images folder (after we've implemented `GetMap`?

Reply:
Yes, we will!